// shopify/dist/local-VTZT5Q4L.js
import {
  alwaysLogAnalytics,
  alwaysLogMetrics,
  analyticsDisabled,
  ciPlatform,
  cloudEnvironment,
  codespacePortForwardingDomain,
  codespaceURL,
  firstPartyDev,
  gitpodURL,
  hasGit,
  homeDirectory,
  isAppManagementDisabled,
  isCloudEnvironment,
  isDevelopment,
  isShopify,
  isTerminalInteractive,
  isUnitTest,
  isVerbose,
  macAddress,
  opentelemetryDomain,
  themeToken,
  useEmbeddedThemeCLI,
  useThemebundling
} from "./chunk-L3FVJFIT.js";
import "./chunk-B5EXYCV3.js";
import "./chunk-75LV6AQS.js";
import "./chunk-57Q6OQG4.js";
import "./chunk-CDQA3SST.js";
import "./chunk-G5R6YD27.js";
import "./chunk-3I3GQNEW.js";
import "./chunk-G2ZZKGSV.js";
import "./chunk-5CLGH5PH.js";
import "./chunk-XE5EOEBL.js";
import "./chunk-5DBS6QOU.js";
import "./chunk-B3YWYURY.js";
import "./chunk-CFIKVUNW.js";
import "./chunk-CDBXAE2F.js";
import "./chunk-PKR7KJ6P.js";
export {
  alwaysLogAnalytics,
  alwaysLogMetrics,
  analyticsDisabled,
  ciPlatform,
  cloudEnvironment,
  codespacePortForwardingDomain,
  codespaceURL,
  firstPartyDev,
  gitpodURL,
  hasGit,
  homeDirectory,
  isAppManagementDisabled,
  isCloudEnvironment,
  isDevelopment,
  isShopify,
  isTerminalInteractive,
  isUnitTest,
  isVerbose,
  macAddress,
  opentelemetryDomain,
  themeToken,
  useEmbeddedThemeCLI,
  useThemebundling
};
